src/target/move.c

```diff
@@ -304,11 +304,19 @@ void turn_back_with_corrections(void)
 {
 	enum step_direction side;
 
-	keep_front_wall_distance(CELL_DIMENSION / 2., 0.001);
 	side = choose_random_side();
```

Comment:
delete it

Reply:
f915452351423a7f9862bc61e4007b0ec62f7dc8